src/js/models/v2/awards/transactions/CoreTransaction.js

```diff
@@ -19,6 +19,7 @@ const CoreTransaction = {
         this._actionTypeDescription = data.actionTypeDescription || '';
         this.modificationNumber = data.modificationNumber || '';
         this.description = data.description || '';
+        this.cfdaNumber = data.cfda_number || '';
```

Comment:
Could you add a test for this?

Reply:
Sure. Test was added in CoreTransaction-test.js.